### PR TITLE
Add Zap Fitness, Bennetts Petroleum, Metro Tas

### PR DESCRIPTION
**Pull request by chuqtas**

### CODE REVIEW

File: data/brands/leisure/fitness_centre.json [hunk 1]
```diff
@@ -714,6 +714,16 @@
         "name": "Xercise4Less"
       }
     },
+    {
+      "displayName": "Zap Fitness",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Zap Fitness",
+        "brand:wikidata": "Q106119874",
```
Comment (LaoshuBaby) [inline data/brands/leisure/fitness_centre.json:722]: If it have a wikipedia pages, you can add it

Comment (LaoshuBaby) [reply data/brands/leisure/fitness_centre.json:722]: Sorry, Just searched, there no wikipedia exist, so this is OK

**Comment by LaoshuBaby** [comment]
Good! Thanks to @chuqtas  for adding those brand!

Next time you can try to run ```npm run build``` if possible, it will generate a unique id for every brand.

**Comment by chuqtas** [comment]
Thanks @LaoshuBaby - yeah I had the conversation about generating the id's before with @bhousel - I do all my editing online and don't have an appropriate dev environment.

**Comment by LaoshuBaby** [comment]
Ah, it’s okay, it’s understandable, ```npm run build``` has been executed just now, and I will run manually for your commit as much as possible.

**Pull request merged**